src/bigquery.ts

```diff
@@ -2232,6 +2232,14 @@ export class BigQuery extends Service {
         job!.getQueryResults(options, callback as QueryRowsCallback);
         return;
       }
+      // If timeout override was provided, return error.
+      if (queryReq.timeoutMs) {
+        const err = new Error(
+          `The query did not complete before ${queryReq.timeoutMs}ms`
```

Comment:
```suggestion
          `The query did not complete before exceeding the ${queryReq.timeoutMs}ms timeout`
```

phrasing nit

Reply:
Similar comment as above, this is replicating the same behavior of `jobs.getQueryResults` and I just copied the message from there, I wonder if someone is depending on exactly that message to handle errors.

https://github.com/googleapis/nodejs-bigquery/blob/ebb3bcda5dbc316626834d1331a51ae41bbb5632/src/job.ts#L613